integration-tests/cli/custom-objects-export.it.js

```diff
@@ -0,0 +1,83 @@
+import fs from 'mz/fs'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/custom-objects-exporter/package.json'
+import {
+  createdCustomObjects,
+  expectedCustomObjects,
+} from './helpers/custom-objects-export.data'
+import { createData, clearData } from './helpers/utils'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'custom-objects-export-int-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('Product Exporter', () => {
+  let apiConfig
+  const bin = './integration-tests/node_modules/.bin/custom-objects-exporter'
+
+  beforeAll(async () => {
+    const credentials = await getCredentials(projectKey)
+    apiConfig = {
+      host: 'https://auth.sphere.io',
+      apiUrl: 'https://api.sphere.io',
+      projectKey,
+      credentials,
+    }
+    await clearData(apiConfig, 'customObjects')
+
+    await createData(apiConfig, 'customObjects', createdCustomObjects)
+  }, 30000)
+
+  afterAll(async () => {
+    await clearData(apiConfig, 'customObjects')
+  })
+
+  describe('CLI basic functionality', () => {
+    it('should print usage information given the help flag', async () => {
+      const [stdout, stderr] = await exec(`${bin} --help`)
+      expect(stderr).toBeFalsy()
```

Comment:
I'd rather be explicit over `.toBeFalsy()` to find out the end value

```js
.toBe(undefined|null|false)
```

Reply:
This one actually returns an empty string. I think `.toBeFalsy()` is a bit prettier than `.toBe('')`, to be honest.

Reply:
yeah I guess it's prettier. in the end, it's important to understand what the expected value can be and that is conveyed through tests.

without understandable tests, you have a hard-to-understand implementation.
I just mention this since there is no guarantee that we maintain this everyday in conjunction to easing onboarding new members to OSS

just food for thought. 😉

Reply:
Depends on your intend. `toBeFalsy` actually means you implementation doesn't care if it's `null`, `undefined`, `''` etc. If you typings however don't even allow that the test ain't really inline with that.

Reply:
In this case, I think `.toBeFalsy()` makes more sense as we don't actually care what the error is, as long as it isn't a truthy value, it'll be fine